src/platform_impl/windows/event_loop.rs

```diff
@@ -197,7 +218,11 @@ impl Default for PlatformSpecificEventLoopAttributes {
 pub struct EventLoopWindowTarget<T: 'static> {
     thread_id: u32,
     thread_msg_target: HWND,
-    pub(crate) runner_shared: EventLoopRunnerShared<T>,
+    pub(crate) runner_shared: EventLoopRunnerShared<UserEventPlaceholder>,
+    // TODO
+    // eventually should be removed after all the backends refactored,
+    // but for now should this be invariant or contra-variant to T?
+    _marker: PhantomData<*mut T>,
```

Comment:
The correct variance and such is already ensured by the top-level `EventLoopWindowTarget`, just use `PhantomData<T>` here.

Reply:
thanks for pointing out. will do.